06-TimeManagement/time_mod/time_mod.c

```diff
@@ -0,0 +1,199 @@
+/*
+ * time_mod.c  CPU load calc module
+ *
+ * Copyright (C) 2019 Chekanov. All Rights reserved.
+ * This program is free software.
+ *
+ */
+
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/err.h>
+#include <linux/rtc.h>
+#include <linux/time.h>
+
+#define ARR 600
+#define BUF 81
```

Comment:
Is 81 enough for all possible value of parsed line?

Reply:
Rewrited all task